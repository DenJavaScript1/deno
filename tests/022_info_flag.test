args: --info http://127.0.0.1:4545/tests/003_relative_import.ts
output: tests/022_info_flag.out
