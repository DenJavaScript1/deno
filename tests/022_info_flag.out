local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
type: TypeScript
compiled: [WILDCARD].js
map: [WILDCARD].js.map
deps:
http://127.0.0.1:4545/tests/019_media_types.ts
  ├── http://localhost:4545/tests/subdir/mt_text_typescript.t1.ts
  ├── http://localhost:4545/tests/subdir/mt_video_vdn.t2.ts
  ├── http://localhost:4545/tests/subdir/mt_video_mp2t.t3.ts
  ├── http://localhost:4545/tests/subdir/mt_application_x_typescript.t4.ts
  ├── http://localhost:4545/tests/subdir/mt_text_javascript.j1.js
  ├── http://localhost:4545/tests/subdir/mt_application_ecmascript.j2.js
  ├── http://localhost:4545/tests/subdir/mt_text_ecmascript.j3.js
  └── http://localhost:4545/tests/subdir/mt_application_x_javascript.j4.js


local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
type: TypeScript
compiled: [WILDCARD].js
map: [WILDCARD].js.map
deps:
http://127.0.0.1:4545/tests/019_media_types.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_typescript.t1.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_video_vdn.t2.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_video_mp2t.t3.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_x_typescript.t4.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_javascript.j1.js
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_ecmascript.j2.js
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_ecmascript.j3.js
\xe2\x94\x94\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_x_javascript.j4.js


local: [WILDCARD]deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
type: TypeScript
compiled: [WILDCARD].js
map: [WILDCARD].js.map
deps:
http://127.0.0.1:4545/tests/019_media_types.ts
http://localhost:4545/tests/subdir/mt_text_typescript.t1.ts
http://localhost:4545/tests/subdir/mt_video_vdn.t2.ts
http://localhost:4545/tests/subdir/mt_video_mp2t.t3.ts
http://localhost:4545/tests/subdir/mt_application_x_typescript.t4.ts
http://localhost:4545/tests/subdir/mt_text_javascript.j1.js
http://localhost:4545/tests/subdir/mt_application_ecmascript.j2.js
http://localhost:4545/tests/subdir/mt_text_ecmascript.j3.js
http://localhost:4545/tests/subdir/mt_application_x_javascript.j4.js


local: /Users/biwanczuk/Library/Caches/deno/deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
type: TypeScript
map: /Users/biwanczuk/Library/Caches/deno/gen/http/127.0.0.1_PORT4545/tests/019_media_types.js.map
deps:
http://127.0.0.1:4545/tests/019_media_types.ts
http://localhost:4545/tests/subdir/mt_text_typescript.t1.ts
http://localhost:4545/tests/subdir/mt_video_vdn.t2.ts
http://localhost:4545/tests/subdir/mt_video_mp2t.t3.ts
http://localhost:4545/tests/subdir/mt_application_x_typescript.t4.ts
http://localhost:4545/tests/subdir/mt_text_javascript.j1.js
http://localhost:4545/tests/subdir/mt_application_ecmascript.j2.js
http://localhost:4545/tests/subdir/mt_text_ecmascript.j3.js
http://localhost:4545/tests/subdir/mt_application_x_javascript.j4.js


 
local: /Users/biwanczuk/Library/Caches/deno/deps/http/127.0.0.1_PORT4545/tests/019_media_types.ts
type: TypeScript
deps:
http://127.0.0.1:4545/tests/019_media_types.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_typescript.t1.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_video_vdn.t2.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_video_mp2t.t3.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_x_typescript.t4.ts
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_javascript.j1.js
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_ecmascript.j2.js
\xe2\x94\x9c\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_text_ecmascript.j3.js
\xe2\x94\x94\xe2\x94\x80\xe2\x94\x80 http://localhost:4545/tests/subdir/mt_application_x_javascript.j4.js
